docs/features/multiple-networks/multi-homing.md

```diff
@@ -232,6 +240,75 @@ spec:
 - specifying a static IP address for the pod is only possible when the
   attachment configuration does **not** feature subnets.
 
+## Persistent IP addresses for virtualization workloads
```

Comment:
might be a naive question.. can this be a subsection feature in itself: https://ovn-kubernetes.io/features/multiple-networks/multi-homing/ ?
I remember breaking out MultiNetworkPolicies...

Reply:
yeah that is fine to do it as part of refactor..